projects/angular/src/data/datagrid/datagrid-row.spec.ts

```diff
@@ -390,6 +388,23 @@ export default function (): void {
         expect(selectionProvider.current).toEqual([]);
       });
 
+      it('verifies a label is not nested within a label when `rowSelectionMode` is enabled', function () {
+        selectionProvider.selectionType = SelectionType.Single;
+        context.testComponent.item = { id: 1 };
+        context.detectChanges();
+        const row = context.clarityElement;
+        selectionProvider.rowSelectionMode = true;
+        expect(row.querySelectorAll('label label').length).toBe(0);
```

Comment:
Thoughts on shortening/simplifying this to `expect(row.querySelector('label label')).toBeNull();`?